pkg/controllers/management/rkeworkerupgrader/plan.go

```diff
@@ -72,6 +74,11 @@ func (uh *upgradeHandler) workerPlan(node *v3.Node, cluster *v3.Cluster) (*v3.RK
 	hostAddress := node.Status.NodeConfig.Address
 	hostDockerInfo := infos[hostAddress]
 
+	if hostDockerInfo.Driver == "" || hostDockerInfo.DockerRootDir == "" {
```

Comment:
What's the workflow in here if error is returned?

Reply:
it would return back to the sync, and keep retrying till the docker info gets assigned